src/applications/gi/components/profile/EstimateYourBenefitsForm.jsx

```diff
@@ -542,124 +584,116 @@ class EstimateYourBenefitsForm extends React.Component {
     const value = shouldRenderEnrolled ? enrolledValue : enrolledOldValue;
 
     return (
+      <Dropdown
+        label={this.renderLearnMoreLabel({
+          text: 'Enrolled',
+          modal: 'calcEnrolled',
+          ariaLabel: ariaLabels.learnMore.calcEnrolled,
+        })}
+        name={name}
+        alt="Enrolled"
+        options={options}
+        visible
+        value={value}
+        onChange={this.handleInputChange}
+        onFocus={this.handleEYBInputFocus}
+      />
+    );
+  };
+
+  renderCalendar = () => {
+    if (!this.props.displayedInputs.calendar) return null;
+
+    const dependentDropdowns = (
       <div>
         <Dropdown
-          label={this.renderLearnMoreLabel({
-            text: 'Enrolled',
-            modal: 'calcEnrolled',
-            ariaLabel: ariaLabels.learnMore.calcEnrolled,
-          })}
-          name={name}
-          alt="Enrolled"
-          options={options}
+          label="How many terms per year?"
+          name="numberNontradTerms"
+          alt="How many terms per year?"
+          options={[
+            { value: '3', label: 'Three' },
+            { value: '2', label: 'Two' },
+            { value: '1', label: 'One' },
+          ]}
+          visible
+          value={this.props.inputs.numberNontradTerms}
+          onChange={this.handleInputChange}
+          onFocus={this.handleEYBInputFocus}
+        />
+        <Dropdown
+          label="How long is each term?"
+          name="lengthNontradTerms"
+          alt="How long is each term?"
+          options={[
+            { value: '1', label: '1 month' },
+            { value: '2', label: '2 months' },
+            { value: '3', label: '3 months' },
+            { value: '4', label: '4 months' },
+            { value: '5', label: '5 months' },
+            { value: '6', label: '6 months' },
+            { value: '7', label: '7 months' },
+            { value: '8', label: '8 months' },
+            { value: '9', label: '9 months' },
+            { value: '10', label: '10 months' },
+            { value: '11', label: '11 months' },
+            { value: '12', label: '12 months' },
+          ]}
```

Comment:
If you wanted you could import this:

https://github.com/department-of-veterans-affairs/vets-website/blob/6a27450abed0eb724d9fa0334633ff8ffe700f78/src/platform/static-data/options-for-select.js#L41-L54

And then do something like:


```js
months.map(({ value, label }) => ({ value, label: `${value} months` }));
```

Reply:
The question the user is answering is "How long is each term?" so the values are not directly related to a specific month of the year.